<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectDialog</class>
 <widget class="QDialog" name="ProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtAda | Configure Project</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/project/project.qrc">
    <normaloff>:/project/app.png</normaloff>:/project/app.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="projectLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="projectIcon">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="projectLabel">
       <property name="text">
        <string>Project Name</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="projectSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="projectEdit">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="placeholderText">
      <string>Enter Project Name...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="firstLine">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="lineWidth">
      <number>6</number>
     </property>
     <property name="midLineWidth">
      <number>6</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="opaqueResize">
      <bool>true</bool>
     </property>
     <property name="handleWidth">
      <number>6</number>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="autLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="autHeaderLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="autIcon">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="autLabel">
           <property name="text">
            <string>Applications Under Test</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="autSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="addAut">
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="toolTip">
            <string>Add New AUT</string>
           </property>
           <property name="styleSheet">
            <string notr="true">padding: 0;</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/project/project.qrc">
             <normaloff>:/project/add.svg</normaloff>:/project/add.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="clearAut">
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="toolTip">
            <string>Add New AUT</string>
           </property>
           <property name="styleSheet">
            <string notr="true">padding: 0;</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/project/project.qrc">
             <normaloff>:/project/clear.svg</normaloff>:/project/clear.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeView" name="autView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="tsLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="tsHeaderLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="tsIcon">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="tsLabel">
           <property name="text">
            <string>Test Scripts</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="tsSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="addTs">
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="toolTip">
            <string>Add New AUT</string>
           </property>
           <property name="styleSheet">
            <string notr="true">padding: 0;</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/project/project.qrc">
             <normaloff>:/project/add.svg</normaloff>:/project/add.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="clearTs">
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="toolTip">
            <string>Add New AUT</string>
           </property>
           <property name="styleSheet">
            <string notr="true">padding: 0;</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/project/project.qrc">
             <normaloff>:/project/clear.svg</normaloff>:/project/clear.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeView" name="tsView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="Line" name="secondLine">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/project/project.qrc"/>
 </resources>
 <connections/>
</ui>
